x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/use_signal_helpers.tsx

```diff
@@ -85,6 +85,7 @@ export const useSignalHelpers = (): {
           // first time signals is defined and validated in the sourcerer
           // redo indexFieldsSearch
           indexFieldsSearch({ dataViewId: sourcererDataView.defaultDataView.id });
+          // indexFieldsSearch({ dataViewId: sourcererDataView.alertDataView.id });
```

Comment:
added to account for index search but commenting out has no effect...

Reply:
What do you mean by `commenting out has no effect`? Do we still need this? Or do we need to understand why it has no effect?